build.gradle

```diff
@@ -87,3 +88,15 @@ tasks.dokkaHtmlMultiModule.configure {
     outputDirectory.set(file("docs/" + project.property("VERSION_NAME")))
     includes.from("README.md")
 }
+
+apply plugin: 'kover'
+
+koverMerged {
+    enable()
+    filters {
+        projects {
+            // Specifies the projects excluded from the merged tasks
+            excludes.addAll(":api-tester", ":integration-tests", ":examples:purchase-tester")
```

Comment:
It would complain I need to enable kover in all these projects if I don't disable them here